### PR TITLE
nmap: add example for scanning live hosts on a network.

### PR DESCRIPTION
**Pull request by jottr**
----

- [ ] The page (if new), does not already exist in the repo.

- [ ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/nmap.md [hunk 1]
```diff
@@ -30,3 +30,7 @@
 - Perform TLS cipher scan against a host to determine supported ciphers and SSL/TLS protocols:
 
 `nmap --script ssl-enum-ciphers {{address_or_addresses}} -p 443`
+
+- List live hosts with hostnames on a specific network:
+
+`nmap -sn 10.0.0.1/24`
```
Comment (pxgamer) [inline pages/common/nmap.md]: Is this different to the first example?

**Comment by agnivade** [comment]
We already have a -sn example.

**Comment by jottr** [comment]
True.
I overlooked the other example. 
Imho, I find my example to be clearer than: `nmap -sn ip_or_hostname optional_another_address` . 
Following that example I would do: `nmap -sn 192.168.0.1` instead of  `nmap -sn 192.168.0.1/24`

**Comment by agnivade** [comment]
Sure, in that case I would ask you to change all the other examples to be ip addresses too. We always prefer concrete values rather than generic text like ip_address or hostname.

File: pages/common/nmap.md (hunk 1 above)
Comment (mebeim) [reply pages/common/nmap.md]: It's actually the same as the first example. This PR should be closed.

**Comment by mebeim** [comment]
Closing this since the added example is equal to the first one.

**Pull request closed without merging**